### PR TITLE
Web Development Courses

### PR DESCRIPTION
**Pull request by Fayob**
# Web Development

- add linters
- add html file for the website building block
- add styles to beautify the mark up pages
- add javascript for DOM manipulation
- ignore some files and folder in .gitignore
- add descriptive README file
- add MIT licence

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+GitHub: [@fayob](https://github.com/fayob)
+
+Twitter: [@fabimworld2536](https://twitter.com/Fabimworld2536)
+
+LinkedIn: [@abimbola-ade](https://www.linkedin.com/in/abimbola-ade)
+
+## Live Demo
+
+[my_portfolio](https://fayob.github.io/portfolio-setup-project/)
+
```
Comment (omarsalem7) [inline README.md]: - [ ] kindly add a live demo link for your project and also kindly add a loom video for presenting your project.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+## Contributing
+
+Contributions, issues, and feature requests are welcome!
+Feel free to check the [issues page](#).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
```
Comment (omarsalem7) [inline README.md:63]: - [ ] Kindly add an "Acknowledgements" section or similar that gives appropriate credit to [Cindy Shin](https://www.behance.net/adagio07) the [author of the original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), as required by the [Creative Commons License](https://creativecommons.org/licenses/).

File: about.html [hunk 3]
```diff
@@ -0,0 +1,225 @@
...
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+    />
+    <link rel="stylesheet" href="/css/main.css" />
+  </head>
+  <body>
+    <nav class="navbar">
+      <ul class="navbar_lists">
+        <li class="navbar_list">
+          <a href="#"><i aria-hidden="true"></i></a>
```
Comment (omarsalem7) [inline about.html]: - [ ] Kindly add a hamburger icon on the about page also for the mobile version.

File: main.html [hunk 4]
```diff
@@ -0,0 +1,365 @@
...
+          <h2>Welcome to Fayob online learning platform</h2>
+        </div>
+
+        <p class="header_paragraph">
+          Lorem ipsum dolor sit amet consectetur adipisicing elit. Similique
+          corrupti tenetur aspernatur repellat ex assumenda omnis, ducimus
+          provident, accusamus deserunt culpa soluta a, illo laborum voluptas
+          eaque dolores sint minus.
+        </p>
+      </header>
```
Comment (omarsalem7) [inline main.html]: - [ ] Kindly align the text to the left and reduce the width of `header_paragraph` like the one of Behance.

File: main.html [hunk 5]
```diff
@@ -0,0 +1,365 @@
...
+          class="partnership_img"
+        />
+        <img
+          src="./assets/microsoft-logoo.jpg"
+          alt="microsoft logo"
+          width="200px"
+          height="100px"
+          class="partnership_img"
+        />
+      </div>
```
Comment (omarsalem7) [inline main.html]: - [ ] kindly remove the margin to match it like the Behance and also remove the open footer tag before the partner section and add it after for the footer part.

File: main.html [hunk 6]
```diff
@@ -0,0 +1,365 @@
...
+          ></a>
+        </li>
+        <li class="footer_list">
+          <a href="#" class="footer_list_a"
+            ><i class="fa fa-whatsapp" aria-hidden="true"></i
+          ></a>
+        </li>
+      </ul>
+      <p class="footer_paragraph">&copy; 2022 Fayob inc. All right reserved</p>
+    </footer>
```
Comment (omarsalem7) [inline main.html]: - [ ] Kindly make a footer like one in the design

 | Your design  | Behance design|
| ------------- | ------------- |
| image  | image|

File: about.html [hunk 7]
```diff
@@ -0,0 +1,225 @@
...
+      </header>
+      <article class="our_story_article">
+        <p class="story_paragraph">
+          Lorem, ipsum dolor sit amet consectetur adipisicing elit. Eius
+          eveniet similique nesciunt accusamus est minus illo culpa
+          temporibus? Obcaecati mollitia consequatur totam architecto
+          officiis, cum nostrum voluptates maxime laboriosam possimus
+          assumenda eos est neque deleniti adipisci ratione eaque hic incidunt
+          libero esse vero. Possimus ipsa ratione sequi deleniti blanditiis
+          libero? Lorem ipsum dolor sit amet consectetur, adipisicing elit. Excepturi cum reiciendis quasi in voluptas delectus rerum quaerat? Debitis, magnam quaerat!
```
Comment (omarsalem7) [inline about.html]: - [ ] Kindly make your design match the one in the Behance by adding the following sections like in the image.
image

**Review by omarsalem7 — CHANGES_REQUESTED** [review]
Hi @Fayob 😊, 

Good job so far! 👏
You have done well reaching this point of the milestone, but there are some issues that you still need to work on to go to the next project but you are almost there! 

you-are-near

## Highlights
- There is no linter error present ✔️
- The Pull Request has a proper title and description. ✔️

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: about.html (hunk 7 above)
Comment (Fayob) [reply about.html]: @omarsalem7, I designed a Web Development website  so I actually think this about page design doesn't match web development platform design. That's why I did it that way

Comment (omarsalem7) [reply about.html]: Hi @Fayob, I agree with you but you should follow the guidelines which you need to follow, and for my work I have guidelines also I need to check if you follow them. I suggest you can add a part for past summits, you can add past two images for students who take a course online or offline. Moreover, you can adjust the content as you want but follow the same design.

File: README.md [hunk 8]
```diff
@@ -0,0 +1,66 @@
...
+GitHub: [@fayob](https://github.com/fayob)
+
+Twitter: [@fabimworld2536](https://twitter.com/Fabimworld2536)
+
+LinkedIn: [@abimbola-ade](https://www.linkedin.com/in/abimbola-ade)
+
+## Live Demo
+
+[click here to see the live demo](https://fayob.github.io/html-css-js-capstone-project/)
```
Comment (Kossi-stack) [inline README.md:45]: - [ ] Kindly fix the live demo link, it is not working.

File: about.html [hunk 9]
```diff
@@ -0,0 +1,234 @@
...
+        </p>
+        <div class="story_image">
+          <img src="./assets/learnining-vision.jpg" alt="learning vision" class="story_image_img">
+          <img src="./assets/e-learning-2.jpg" alt="e-learning" class="story_image_img">
+        </div>
+      </article>
+    </section>
+    <section>
```
Comment (Kossi-stack) [inline about.html:194]: - [ ] Kindly fix these two images, put an overlay color on the image and add text as well.

-  Your Design 

Screen Shot 2022-09-15 at 09 25 19
-  [ ] Behance

Screen Shot 2022-09-15 at 09 51 17

**Review by Kossi-stack — CHANGES_REQUESTED** [review]
## Hi @,

**Good job so far! 👍🏻**
There are some issues that you still need to work on to go to the next project but you are almost there!💪

68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f556461535559436c723932636a57706556782f67697068792e676966

## To Highlight

✅ Descriptive pull request message
✅ linter error checks validated
✅ Documented your work professionally
✅ README file is descriptive.
✅ Commit messages are meaningful.

## Required Changes ♻️

 _Check the comments under the review for more changes_.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

_**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by BertrandConxy — APPROVED** [review]
Hi @Fayob , 

In my opinion, your project is ready for the presentation!  There is nothing else to say other than... it's time to merge it :shipit: 

## Highlights 💪 
- no linter errors
- project requirements are met
- good PR description
- good README


Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**